lighthouse-core/config/config.js

```diff
@@ -294,9 +285,9 @@ class Config {
     this._audits = configJSON.audits ? expandAudits(
         filterAudits(configJSON.audits, auditWhitelist), this._configDir
         ) : null;
-    // filterPasses expects audits to have been expanded
+    // validatePasses expects audits to have been expanded
     this._passes = configJSON.passes ?
-        filterPasses(configJSON.passes, this._audits, this._configDir) :
+        validatePasses(configJSON.passes, this._audits, this._configDir) :
```

Comment:
this was also serving to make a copy of the `passes` array and the individual passes. We don't want to modify the config passed into us (for use as a node module and for the extension)